src/style.css

```diff
@@ -0,0 +1,153 @@
+.items {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid rgb(155, 145, 145);
+}
+
+.list-items h3 {
+  text-align: left;
+  padding-bottom: 2px;
+}
+
+.list-items {
+  width: 400px;
+  justify-content: center;
+  list-style: none;
+  text-align: left;
+  background-color: white;
+  float: right;
+  z-index: 999;
+}
```

Comment:
I noticed some redundant styling in your code. The .list-items class has the float: right; property, but there is no need for it since it is not floated alongside any other elements. Kindly, you can remove the float: right; style from the .list-items class.😊